discord/__init__.py

```diff
@@ -22,10 +22,10 @@
 
 from nextcord import *
 
-__title__ = 'discord'
+__title__ = 'nextcord'
 __author__ = 'tag-epic & Rapptz'
 __license__ = 'MIT'
-__copyright__ = 'Copyright 2015-present Rapptz'
+__copyright__ = 'Copyright 2015-present Rapptz & tag-epic'
```

Comment:
```suggestion
__copyright__ = 'Copyright 2015-2021 Rapptz & 2021-present tag-epic'
```
at least this is what i think, as epic didnt have it in 2015 and rapptz doesnt have it now